src/GovUk.Education.ExploreEducationStatistics.Content.Services/ReleaseService.cs

```diff
@@ -24,44 +25,50 @@ public class ReleaseService : IReleaseService
     {
         private readonly IPersistenceHelper<ContentDbContext> _persistenceHelper;
         private readonly IFileStorageService _fileStorageService;
-        private readonly IMethodologyService _methodologyService;
+        private readonly IMethodologyCacheService _methodologyCacheService;
         private readonly IUserService _userService;
         private readonly IPublicationService _publicationService;
         private readonly IMapper _mapper;
 
         public ReleaseService(
             IPersistenceHelper<ContentDbContext> persistenceHelper,
             IFileStorageService fileStorageService,
-            IMethodologyService methodologyService,
+            IMethodologyCacheService methodologyCacheService,
             IUserService userService,
             IPublicationService publicationService,
             IMapper mapper)
 
         {
             _persistenceHelper = persistenceHelper;
             _fileStorageService = fileStorageService;
-            _methodologyService = methodologyService;
+            _methodologyCacheService = methodologyCacheService;
             _userService = userService;
             _publicationService = publicationService;
             _mapper = mapper;
         }
 
+        // TODO EES-3643 - move into a ReleaseCacheService?
         public async Task<Either<ActionResult, ReleaseViewModel>> GetCachedViewModel(string publicationSlug, string? releaseSlug = null)
```

Comment:
`MethodologyService.GetCachedSummariesByPublication` was moved (renamed as MethodologyCacheService.GetSummariesByPublication`), so on that basis this one should probably be moved too. However there would be a dependency in `ReleaseCacheService` on `MethodologyCacheService` then which is not so nice.

If this isn't moved, then should we revert the other one back? 😅